packages/contentful--app-scripts/lib/open/open-settings.js

```diff
@@ -0,0 +1,44 @@
+const open = require('open');
+const chalk = require('chalk');
+const inquirer = require('inquirer');
+const { APP_DEF_ENV_KEY } = require('../../utils/constants');
+
+const REDIRECT_URL = 'https://app.contentful.com/deeplink?link=app-definition';
+
+async function openSettings(options) {
+  let definitionId;
+  if (options.definitionId) {
+    definitionId = options.definitionId;
+  } else if (process.env[APP_DEF_ENV_KEY]) {
+    definitionId = process.env[APP_DEF_ENV_KEY];
+  } else {
+    const prompts = await inquirer.prompt([
+      {
+        name: 'definitionId',
+        message: `The id of the app:`,
+      },
+    ]);
+    definitionId = prompts.definitionId;
+  }
+
+  console.log(definitionId, 'DEFID');
+  console.log(options, 'options');
+  console.log(process.env[APP_DEF_ENV_KEY], 'process.env[APP_DEF_ENV_KEY]');
+
+  if (!definitionId) {
+    console.log(`
+        ${chalk.red('Error:')} There was no app-definition defined.
+
+        Please add it with ${chalk.cyan('--definition-id=<app-definition-id>')}
+        or set the environment variable ${chalk.cyan(`${APP_DEF_ENV_KEY} = <app-definition-id>`)}
+      `);
+    throw new Error('No app-definition-id');
+  }
+
+  open(`${REDIRECT_URL}&id=${definitionId}`);
```

Comment:
I'd wrap this in a try just in case. We do this in getMgmtToken also

Reply:
Yes I think that makes sense 👍